src/Spryker/Zed/AppPayment/Dependency/Plugin/AppPaymentPlatformMarketplacePluginInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Zed\AppPayment\Dependency\Plugin;
+
+use Generated\Shared\Transfer\PaymentTransmissionsRequestTransfer;
+use Generated\Shared\Transfer\PaymentTransmissionsResponseTransfer;
+
+interface AppPaymentPlatformMarketplacePluginInterface extends AppPaymentPlatformPluginInterface
+{
+    /**
+     * Specification:
+     * - Transfers payments.
+     * - Requires `PaymentTransmissionsRequestTransfer::transactionId`to be set.
+     * - Requires `PaymentTransmissionsRequestTransfer::appConfig`to be set.
```

Comment:
@stereomon Transfers payments from whom to whom?

Aren't there more required fields? Please check where `$paymentTransmissionsRequestTransfer->getPaymentTransmissions()` are set?   There is `$paymentTransmissionTransfer->getMerchantOrFail()`,

https://github.com/spryker-projects/app-stripe/pull/151/files#diff-9659662e9ef1ebe97438634bf0c7dcbeec297c2729c6d5a85ed3d4082cb8661fR52

Reply:
The merchant information is added from the AppMerchant package https://github.com/spryker/app-merchant/blob/main/src/Spryker/Zed/AppMerchant/Business/PaymentTransmissions/PaymentTransmissionsRequestExtender.php

Reply:
> Transfers payments from whom to whom?

Either from the main account to the merchant account or vice versa. I didn't want to add the Merchant Term here

Reply:
so basically the main business logic has to be implemented in the platform plugin implementation. I recommend that it is written in the spec, because from a developer's point of view, it's not clear how to use/implement this method in the plugin.

Reply:
FYI There is also Order Item context and Merchant Reference context in the OpenAPI
- https://github.com/spryker/app-payment/blob/main/src/Spryker/Shared/AppPayment/Transfer/app_payment.transfer.xml#L320C53-L320C72
- https://github.com/spryker/app-payment/blob/main/resources/api/openapi.yml#L249-L252 the PaymentTransmission transfer is not defined in the package.